.gitignore

```diff
@@ -13,6 +13,6 @@ reflex.conf
 generated_ftl_module.go
 *.zip
 dist/
-examples/**/_ftl/
 examples/**/go.work
 examples/**/go.work.sum
+**/_ftl
```

Comment:
The `examples/` will have to stay, in this repo, because we have scaffolding called the same `_ftl`

Actually let's keep this, and rename the folders in the template directories to `_ftl.tmpl` - the scaffolder should rename that correctly, but maybe just verify `ftl build` still works on a Go module.

Reply:
ah nice! updated and validated the build